### PR TITLE
Signup aaron doran

### PR DESCRIPTION
**Pull request by AJWD92**

### CODE REVIEW

File: fe/src/components/Signup/FormUserDetails.jsx [hunk 1]
```diff
@@ -60,20 +87,21 @@ const FormUserDetails = props => {
...
                     margin='normal'
-                    halfWidth
                     variant='contained'
                     label='Continue'
                     style={styles.button}
                     onClick={next}
                 >Continue
                 </Button>
                 </Container>
+                </div>
```
Comment (PaulMEdwards) [inline fe/src/components/Signup/FormUserDetails.jsx]: Please clean-up the indention when you get a chance.

Comment (AJWD92) [reply fe/src/components/Signup/FormUserDetails.jsx]: Okay I dont know why it got so off but I have Prettier on now so it should clean that up but I'll make sure to double check

File: fe/src/components/UserProfile/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    });
+
+    useEffect(() => {
+        axiosWithAuth()
+        .get('https://fish-friends-build-week.herokuapp.com/')
+        .then(res => {
+            console.log(res.data);
+        });
+        
+    })
```
[the remark this answers is not in the document]
Comment (AJWD92) [reply fe/src/components/UserProfile/Profile.jsx]: Good catch

**Comment by AJWD92** [comment]
I actually fixed it let me know if anything else pops up

**Pull request merged**